### ISSUE TITLE
Floccus Android app fails to sync via git

### ISSUE THREAD

**Issue by msoneal0** [floccusaddon/floccus#1631]
### Which version of floccus are you using?

5.1.7

### How many bookmarks do you have, roughly?

112

### Are you using other means to sync bookmarks in parallel to floccus?

No

### Sync method

Git

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Android

### Describe the Bug

Floccus app fails to sync via git claiming that the commit is not available locally.

### Expected Behavior

No error during git sync

### To Reproduce

create/import any git based sync profile and attempt to sync. it doesn't matter if the git repo is new, empty or existing I get the same issue.

### Debug log provided

- [X] I have provided a debug log file

**Comment by msoneal0** [comment]
![Screenshot_20240601-111939](https://github.com/floccusaddon/floccus/assets/171437858/e5ed86ee-4639-4ea6-b18d-39cece83d6d4)
[floccus-5.1.7-2024-06-01-redacted.log](https://github.com/user-attachments/files/15522562/floccus-5.1.7-2024-06-01-redacted.log)

**Comment by marcelklehr** [comment]
This is a duplicate of #1608
